tests/all/test_values.rs

```diff
@@ -525,7 +525,7 @@ fn test_metadata() {
 
 #[test]
 fn test_floats() {
-    #[cfg(not(any(feature = "llvm14-0", feature = "llvm15-0")))]
+    #[cfg(not(feature = "llvm15-0"))]
```

Comment:
Could we put llvm_versions on the function instead?

Reply:
I tried. Doesn't work with tests I guess.

Reply:
Can you fix the CI please?

Reply:
I can't; waiting on https://github.com/KyleMayes/install-llvm-action/issues/31